install.sh

```diff
@@ -121,12 +130,18 @@ else
   echo "[PRE-INSTALL]: Kubeconfig variable is not set. Kubectl will use: ~/.kube/config"
 fi
 
+
+echo "[PRE-INSTALL]: Setting Gateway IP"
+GATEWAY_IP=$(set_gateway_node)
```

Comment:
```suggestion
GATEWAY_IP=set_gateway_node
```

Reply:
Are you sure of this? I got this output:
```
[PRE-INSTALL]: Setting Gateway IP
[PRE-INSTALL]: GATEWAY_IP is set to: set_gateway_node
[PRE-INSTALL]: POD_CIDR is set to: 10.244.0.0/16
```

Reply:
Same as above. No not consider this